ethereum/contracts/SwapFactory.sol

```diff
@@ -19,9 +19,9 @@ contract SwapFactory is ERC2771Context, Secp256k1 {
     }
 
     struct Swap {
-        // contract creator, Alice
+        // individual swap creator, Alice
         address payable owner;
```

Comment:
Not related to package comments. I was just looking at the contract to better understand the relayer and the comment above looked wrong.